src/redisraft.c

```diff
@@ -1265,23 +1363,20 @@ __attribute__((__unused__)) int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisMod
         return REDISMODULE_ERR;
     }
 
+    if (RedisModule_SubscribeToServerEvent(ctx, RedisModuleEvent_Config,
+                                           handleConfigChangeEvent) != REDISMODULE_OK) {
+        RedisModule_Log(ctx, REDIS_WARNING, "Failed to subscribe to server events.");
+        return REDISMODULE_ERR;
+    }
+
     RedisRaftCtx *rr = &redis_raft;
 
 #ifdef HAVE_TLS
     if (rr->config->tls_enabled) {
-         redisSSLContextError ssl_error;
-         rr->ssl = redisCreateSSLContext(rr->config->tls_ca_cert,
-                                         NULL,
-                                         rr->config->tls_cert,
-                                         rr->config->tls_key,
-                                         NULL,
-                                         &ssl_error);
-         if (!rr->ssl) {
-             RedisModule_Log(ctx, REDIS_WARNING, "Failed to create ssl context.");
-             LOG_ERROR("Error: %s", redisSSLContextGetError(ssl_error));
-             return REDISMODULE_ERR;
-         }
+        RedisModule_Log(ctx, REDIS_WARNING, "Generating SSL Context");
+        rr->ssl = generateSSLContext(ctx, rr);
     }
+    RedisModule_Log(ctx, REDIS_WARNING, "ssl context = %p", rr->ssl);
```

Comment:
Leftover? If not, let's use debug level and LOG_DEBUG()

Reply:
leftover, was from debugging.